src/main/java/org/folio/circulation/domain/AccountRepository.java

```diff
@@ -110,4 +114,13 @@ public AccountRepository(Clients clients) {
   private MultipleRecordFetcher<FeeFineAction> createFeeFineActionFetcher() {
     return new MultipleRecordFetcher<>(feefineActionsStorageClient, "feefineactions", FeeFineAction::from);
   }
+
+  public CompletableFuture<Result<Account>> create(JsonObject accountRepresentation) {
```

Comment:
Why does this take JSON as a parameter rather than the `Account` domain class?

Reply:
Fixed.